### DISCUSSION TITLE
How to customize the date range picker cell hover style?

### DISCUSSION THREAD

**Discussion by AsuraKev** [ant-design/ant-design#44186] [Q&A]
Which token in the config provider that allow us to customize the dashed border of the hovered cell when selecting a range in a date range picker? E.g I want to have a background color for the cells that are hovered during a range selection

thanks

![image](https://github.com/ant-design/ant-design/assets/52765572/ddcf24a3-4806-43ab-bb9e-1a2a7216f53c)

**Comment by Vakulenko88** [comment]
For example:
`.ant-picker-cell::after {
  border: none !important;
}`

**Comment by nissandev** [comment]
here you can see the example css
.ant-picker-cell-in-view.ant-picker-cell-range-start .ant-picker-cell-inner {
  background-color: #f26600 !important;
}

.ant-picker-cell-in-view.ant-picker-cell-range-end .ant-picker-cell-inner {
  background-color: #f26600 !important;
}

.ant-picker-cell-in-view.ant-picker-cell-today
  .ant-picker-cell-inner::before {
  border-color: #f26600 !important;
}

/* Override placeholder text color for RangePicker */
.ant-picker .ant-picker-input > input::placeholder {
  color: #999; /* Change to your desired color */
}

.ant-picker-cell-in-view.ant-picker-cell-range-start:not(.ant-picker-cell-range-start-single)::before,
.ant-picker-cell-in-view.ant-picker-cell-in-range::before,
.ant-picker-cell-in-view.ant-picker-cell-range-end:not(.ant-picker-cell-range-end-single)::before,
.ant-picker-time-panel-column
  > li.ant-picker-time-panel-cell-selected
  .ant-picker-time-panel-cell-inner {
  background: #f8dac4 !important;
  
}

.ant-btn-primary {
background-color: #27c456 !important;
border-color: #1f9643e5 !important;
}

**Comment by Arzamendiaariel** [comment]
is there a way to prevent the update of the input's value when we hover through options?
